requirements_base.txt

```diff
@@ -1,7 +1,8 @@
 Pillow==2.6.1
 PyPlatec==1.3.1.post1
-langgen==0.1.2
 argparse==1.2.1
+gitpython==1.0.1
+langgen==0.1.2
```

Comment:
that was a dependancy of gitpython, but this is going away. going to fix this in travis.